dockerfile/rocm5.0-pytorch1.9.0.dockerfile

```diff
@@ -0,0 +1,151 @@
+FROM rocm/pytorch:rocm5.0_ubuntu18.04_py3.7_pytorch_1.9.0
+
+# OS:
+#   - Ubuntu: 18.04
+#   - OpenMPI: 4.0.5
+#   - Docker Client: 20.10.8
+# ROCm:
+#   - ROCm: 5.0.0
+#   - RCCL: 2.10.3
+#   - RCCL RDMA SHARP plugins: rocm-rel-5.0
+#   - hipify: 5.0.0
+# Mellanox:
+#   - OFED: 5.2-2.2.3.0
+#   - HPC-X: v2.8.3
+# Intel:
+#   - mlc: v3.9a
+
+LABEL maintainer="SuperBench"
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN wget -qO - http://repo.radeon.com/rocm/rocm.gpg.key | APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 apt-key add - && \
```

Comment:
Would you please explain why add it back?

Reply:
deleted